<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>USARTAss</class>
 <widget class="QMainWindow" name="USARTAss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySoftware</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_9">
      <property name="minimumSize">
       <size>
        <width>931</width>
        <height>301</height>
       </size>
      </property>
      <property name="title">
       <string>CHart</string>
      </property>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>276</width>
         <height>238</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>276</width>
         <height>238</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>276</width>
         <height>238</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QChartView" name="VDetectGGraph_2"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="VDetectCoord_2">
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;坐标&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>276</width>
         <height>238</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>276</width>
         <height>238</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>276</width>
         <height>238</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QChartView" name="VDetectGGraph_3"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="VDetectCoord_3">
          <property name="minimumSize">
           <size>
            <width>121</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;坐标&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>276</width>
         <height>238</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="VDetectCoord_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>208</y>
          <width>256</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;坐标&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QChartView" name="VDetectGGraph_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_13">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>20</y>
         <width>251</width>
         <height>241</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>231</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Nirmala UI</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QTextEdit" name="Chart2StartFrame">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Nirmala UI'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Agency FB','Nirmala UI','Agency FB'; font-style:italic;&quot;&gt;START2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Chart2StartFrame(String)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Chart1StartFrame(String)</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="Chart1StartFrame">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Nirmala UI'; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Agency FB','Agency FB'; font-weight:400;&quot;&gt;START1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Chart3StartFrame(String)</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="Chart3StartFrame">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Nirmala UI'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Agency FB','Nirmala UI','Agency FB'; font-style:italic;&quot;&gt;START3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SetChartFrame">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="SendSerialMessage">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Send Message</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>721</width>
        <height>348</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTextBrowser" name="RXBytescount">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>701</width>
         <height>41</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>701</width>
         <height>41</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>15</y>
         <width>65</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;RecvSpace&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="RecvSpace">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>701</width>
         <height>192</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>701</width>
         <height>192</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>701</width>
         <height>192</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="ClearSendSpace">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>clear Send Space</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="minimumSize">
       <size>
        <width>701</width>
        <height>131</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>721</width>
        <height>131</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;SendSpace&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="SendSpace">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>481</width>
        <height>229</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Data bits</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Baud rate</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="ParityInfo">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Space</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mark</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="RefreshUSART">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Refresh USART 
information</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="DataBitsInfo">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="StopBitsInfo">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Parity bit</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="OpenCloseUSART">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Stop bit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="USARTInfo">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Select the serial port

</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="BaudInfo">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2" colspan="2">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="OpenfraemCheck">
            <property name="text">
             <string>OpenfraemCheck</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ClosefraemCheck">
            <property name="text">
             <string>ClosefraemCheck</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
